app/code/Magento/InventorySales/Test/Integration/IsProductSalable/MinQtyConditionTest.php

```diff
@@ -108,9 +108,9 @@ public function executeWithManageStockFalseAndMinQty(): array
             ['SKU-2', 10, false],
```

Comment:
Looks there are also issues in:
- \Magento\InventorySales\Test\Integration\IsProductSalable\ManageStockConditionTest
- \Magento\InventorySales\Test\Integration\IsProductSalableForRequestedQty\ManageStockConditionTest